spec/services/bulk_import_row_service_spec.rb

```diff
@@ -100,54 +100,45 @@
       end
 
       shared_examples 'common behavior' do
-        context 'when the target account is already followed' do
-          before do
-            account.follow!(target_account)
+        shared_examples 'row import success and list addition' do
+          it 'returns true and adds the target account to the list' do
+            result = nil
```

Comment:
This is sort of odd looking - but is here because as far as I know you can't both assert something about result of the call (the block here calling `subject.call`) and then also use the block form to be able to use `change` or similar after the fact. This is a small workaround for that, which I think is fine - just wanted to note the style thing.